.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Build target user
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@tracewrap-8.1
+    - name: Copy source.list file to include deb-src
+      run: |
+        sudo cp /etc/apt/sources.list /etc/apt/sources.list.d/tmp.list
+        sudo sed -i "s/# deb-src/deb-src/g" /etc/apt/sources.list.d/tmp.list
+    - name: Install deps
+        sudo apt-get  update
+        sudo apt-get --no-install-recommends -y build-dep qemu
+        sudo apt-get install -y autoconf libtool protobuf-c-compiler
+    - name: Install OCaml
+      uses: ocaml/setup-ocaml@v2
+      with:
+        ocaml-compiler: 4.14.x
+        dune-cache: true
+        opam-disable-sandboxing: true
+    - name: Install piqi
+      run: |
+        opam install piqi
+    - name: Clone qemu and bap-frames
+      run: |
+        git clone http://github.com/BinaryAnalysisPlatform/bap-frames.git
+        git clone http://github.com/BinaryAnalysisPlatform/qemu.git
+    - name: Build target
+      run: |
+        branch=$(git rev-parse --abbrev-ref HEAD)
+        if [ "$branch" == "tracewrap-8.1" ]; then
+          # No build test needed
+          exit 0
+        fi
+        echo "NOTE:"
+        echo "Please update this workflow so it builds the target you are working on."
+        echo "See the commented lines for an example."
+        exit 1
```

Comment:
Okay, I am not getting it, what's the purpose of a workflow that doesn't build the code? Let's at least build it without any targets, or with a target but that is not tracewrapped.

Reply:
My thinking was, that we only want to test the build for targets which have tracing implemented. Since `tracewrap-8.1` has no such target, we just skip it and notify the user to add the build step in the CI for their target.
So for `tracewrap-8.1-hexagon` we only build Hexagon and so forth.

The reason I didn't added a normal build here, was that I did not see the point in wasting CI time, if we do not care about building qemu without targets with tracing.
Does this make sense?

Reply:
This branch contains changes to TCG and other core parts of qemu, so we would like to be at least sure that they do compile. Building with any target, e.g., x86, or maybe without any targets at all (if it builds the core part will check that a change to the code base doesn't break at least the compilation part. 

Otherwise, this CI will just waste time by only installing the prerequisites of the repository, in which I don't see any point.